cloudigrade/account/tasks.py

```diff
@@ -228,3 +235,55 @@ def run_inspection_cluster(messages, cloud='aws'):
         cluster=settings.HOUNDIGRADE_ECS_CLUSTER_NAME,
         taskDefinition=result['taskDefinition']['taskDefinitionArn'],
     )
+
+
+def persist_aws_inspection_cluster_results(inspection_result):
+    """
+    Persist the aws houndigrade inspection result.
+
+    Args:
+        inspection_result (dict): A dict containing houndigrade results
+    Returns:
+        None
+    """
+    RHEL_TAG = ImageTag.objects.filter(description='rhel').first()
+    results = inspection_result.get('results', [])
+    for image_id, image_json in results.items():
+        ami = AwsMachineImage.objects.filter(ec2_ami_id=image_id).first()
+        if ami is not None:
+            ami.tags.clear()
+            rhel_found = any([attribute.get('rhel_found')
+                              for disk_json in list(image_json.values())
+                              for attribute in disk_json.values()])
+            if rhel_found:
+                ami.tags.add(RHEL_TAG)
+            # Add image inspection JSON
+            ami.inspection_json = json.dumps(image_json)
+            ami.save()
+        else:
+            logger.error(
+                _('AwsMachineImage "{0}" is not found.').format(image_id))
+
+
+@shared_task
+def persist_inspection_cluster_results_task():
+    """
+    Task to run periodically and read houndigrade messages.
+
+    Returns:
+        None: Run as an asynchronous Celery task.
+
+    """
+    messages = read_messages_from_queue(
+        settings.HOUNDIGRADE_RESULTS_QUEUE_NAME,
+        HOUNDIGRADE_MESSAGE_READ_LEN)
+    if bool(messages):
+        for message in messages:
+            inspection_result = message
+            if isinstance(message, str):
+                inspection_result = json.loads(message)
+            if inspection_result.get(CLOUD_KEY) == CLOUD_TYPE_AWS:
+                persist_aws_inspection_cluster_results(inspection_result)
+            else:
+                logger.error(_('Unsupported cloud type: "{0}"'.format(
+                               message.get(CLOUD_KEY))))
```

Comment:
Parentheses aren't nested correctly here for formatting the localized string.

Should be:
```
logger.error(_('Unsupported cloud type: "{0}"').format(message.get(CLOUD_KEY))) 
```

Reply:
Yep.  Good catch.